doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,65 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication.
+
+//Are all steps in the appliance console menu?
+
+. Confirm and note the hostnames of the active database (primary) appliance and the standby database appliance in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
+//Is there an easier way than logging into the appliance console of one appliance and verifying if "Local Database Server" says "running (primary)" or "running (standby)"?
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. Stop the failover monitor on the non-database {product-title_short} appliances:
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. Reset the standby by deleting the database:
+//Configure Database Replication > Configure Server as Standby? Then ??
++
+----
+# COMMAND
+----
++
+. Stop `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl stop evmserverd
+----
++
+. Rename the database-only appliance and restart that appliance:
+.. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+.. Restart the renamed appliance.
+//Rename, then restart the standby if desired now?
++
+. On all appliances, change the database to the right hostname in vmdb/config/database.yml:
++
+----
+# COMMAND
+----
++
+. Restart the appliances:
+//Does order of restarting matter? Which appliances?
++
+----
+# COMMAND
+----
++
+[IMPORTANT]
+====
+If you are using non-dedicated database appliances, also stop `evmserverd` on those appliances before renaming it and reconfigure the `database.yml` before restarting.
+====
+
+At this point, all appliances should be able to connect to the database. 
+
+//Add a step for the user to test this.
+
+After that you will have to reconfigure high availability from xref:configuring_primary_db[] 
+//(part 2.3 in Installation - All steps from 2.3 to 2.6?).
```

Comment:
Where is this? Most of this looks okay, I'm concerned that this doesn't cover all the steps that would be needed to get the hostname updated in the internal tracking tables. Can you point out the docs that you're referencing here?

Reply:
Sure, it's in chapter 2 of the published guide on redhat.com --> https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html-single/high_availability_guide/#configuring_primary_db

It's a bit repetitive, but for a better UX, I'd rather re-list the steps the user needs here, rather than make them flip backward in the guide. But if it's completely identical to sections 2.3-2.6, we could potentially do that instead. What do you think?

Reply:
I'm a bit concerned that re-running the primary configuration through the console will fail if they use the same node number. Can you ensure that someone tested these steps?

Reply:
Also it may be worth noting here that there will be a bunch of confirmation messages when re-configuring a standby that has data and was previously configured for HA.

Reply:
Ah and one last thing. I don't thin step 2.6 (about installing additional appliances) is really needed here. In this case we're re-configuring the already installed appliances by editing the `database.yml` file.

Reply:
Thanks Nick! I've reorganized the content a little with your info above, but agree, definitely need these steps to be tested. I'm asking QE for that, and will keep you updated.